rules-tests/Php80/Rector/ClassMethod/AddParamBasedOnParentClassMethodRector/Source/ValuesTraitAbstract.php

```diff
@@ -1,10 +0,0 @@
-<?php
-
-namespace Rector\Tests\Php80\Rector\ClassMethod\AddParamBasedOnParentClassMethodRector\Source;
-
-trait ValuesTraitAbstract
-{
-    private array $values = [];
-
-	abstract public function addValue(int $value);
```

Comment:
this trait method is abstract, actually need to be implemented by trait that consume it.

That's why there is skip trait that not implements abstract trait fixture, and trait fixture that override this abstract method.